sparkplug-core/src/clojure/sparkplug/rdd.clj

```diff
@@ -188,11 +190,7 @@
    (HashPartitioner. n))
   ^HashPartitioner
```

Comment:
Should the return type on this still be `HashPartitioner`? It looks like `FnHashPartitioner` extends just `Partitioner`

Reply:
Nice catch!